Demo/src/main/java/com/paypal/android/DemoActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.paypal.android
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.paypal.android.card.payWithCard
+import com.paypal.android.core.PaymentsClient
+
+class DemoActivity : AppCompatActivity() {
```

Comment:
What are everyone's thoughts on creating a kotlin specific demo and a java specific demo? If we have the bandwidth, it'd be nice to show a java integration along with the ability to test java syntax for new sdk surface area.

Reply:
I think that's a good idea. It'll be a good playground to write sample code as well.